includes/class-woocommerce.php

```diff
@@ -844,6 +854,15 @@ public function mailer() {
 		return WC_Emails::instance();
 	}
 
+	/**
+	 * Theme support class.
+	 *
+	 * @return ThemeSupport
+	 */
+	public static function theme_support() {
```

Comment:
As we discussed in Slack, I think we might benefit from introducing a dependency container now, as opposed to further extending this global.

Reply:
On it!